lib/imagec/docker.go

```diff
@@ -335,8 +349,12 @@ func FetchImageManifest(ctx context.Context, options Options, schemaVersion int,
 	if err != nil {
 		return nil, "", err
 	}
-	url.Path = path.Join(url.Path, options.Image, "manifests", options.Tag)
 
+	manifestRef := options.Tag
+	if digested, ok := options.Reference.(reference.Canonical); ok {
```

Comment:
Seems a little D.R.Y... Any reason why you can't just call `manifestRef := tagOrDigest(options.Reference, options.Tag)`?